db/models.py

```diff
@@ -50,3 +60,57 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=CASCADE)
```

Comment:
The code style issue: you are using the `settings.AUTH_USER_MODEL` directly. It's better to use the `get_user_model()` method to get the user model. This method will always return the currently active user model – the custom user model if one is specified, or `User` otherwise.